packages/typescript-configs/README.md

```diff
@@ -55,10 +54,24 @@ Here is an example of a tsconfig.json file extending the Workleap TypeScript con
 
 ```json
 {
-  "extends": "@workleap/typescript-configs/monorepo-root"
+  "extends": "@workleap/typescript-configs/monorepo-root",
+  "exclude": ["packages", "node_modules"]
 }
 ```
 
+**Note**: If you have a tsconfig.json file in your monorepo root and you exclude all packages, you might get a TS18003 error because TypeScript can't find any files to lint. To fix this, you can either add an empty `references` property to your tsconfig.json file:
```

Comment:
There's a typo, you added an "either" but you only give a single choice.

Either add a second choices or remove the "either" word :)

Reply:
i move the delete under the example and forgot the either :awkward:. Thanks